pkg/pgmodel/model/labels.go

```diff
@@ -9,23 +9,50 @@ import (
 	"fmt"
 	"math"
 	"sort"
+	"strconv"
 	"strings"
 	"sync"
 
 	"github.com/prometheus/prometheus/pkg/labels"
 	"github.com/timescale/promscale/pkg/prompb"
 )
 
+// SeriesID represents a globally unique id for the series. This should be equivalent
+// to the PostgreSQL type in the series table (currently BIGINT).
+type SeriesID int64
+
+const unsetSeriesID = -1
```

Comment:
can we have this as `defaultSeriesID`? unsetSeriesID makes me sound like a function that unsets the seriesID, which is unrelated.

Reply:
It's an invalid seriesID, shouldn't be called default

Reply:
I agree, but something like initSeriesID then?

Reply:
renaming to invalidSeriesID